CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
+ * Error range for `src-import.1.2`. See[#499](https://github.com/angelozerr/lsp4xml/issues/499)
+ * Error range for `s4s-elt-invalid-content.3`. See[#496](https://github.com/angelozerr/lsp4xml/issues/496)
+ * NPE in `textDocument/definition` in XSD files. See[#488](https://github.com/angelozerr/lsp4xml/issues/488)
+ * Error range for `cvc-pattern-valid`. See[#477](https://github.com/angelozerr/lsp4xml/issues/477)
+ * Error range for `AttributePrefixUnbound`. See[#476](https://github.com/angelozerr/lsp4xml/issues/476)
+ * Error range for `EmptyTargetNamespace`. See[#472](https://github.com/angelozerr/lsp4xml/issues/472)
+ * Error range for `ct-props-correct.3`. See[#467](https://github.com/angelozerr/lsp4xml/issues/467)
+ * Error range for `sch-props-correct.2`. See[#462](https://github.com/angelozerr/lsp4xml/issues/462)
+ * Error range for `s4s-elt-must-match.2`. See[#458](https://github.com/angelozerr/lsp4xml/issues/458)
+ * Error range for `ct-props-correct.3`. See[#455](https://github.com/angelozerr/lsp4xml/issues/455)
+ * Error range for `src-ct.1`. See[#453](https://github.com/angelozerr/lsp4xml/issues/453)
+ * Error range for `duplicate attribute`.. See[#452](https://github.com/angelozerr/lsp4xml/issues/452)
+ * Error range for `p-props-correct.2.1`. See[#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Element/Attribute names must be case sensitive. See[#433](https://github.com/angelozerr/lsp4xml/issues/433)
+ * Selection formatting ignores attribute indentation preference. See[#429](https://github.com/angelozerr/lsp4xml/issues/429)
+ * Error range for `cos-all-limited.2`. See[#428](https://github.com/angelozerr/lsp4xml/issues/428)
+ * Error range for `src-element.3`. See[#420](https://github.com/angelozerr/lsp4xml/issues/420)
+ * Documents with an Internal Subset DOCTYPE shouldn't stop trying to bind. See[#379](https://github.com/angelozerr/lsp4xml/issues/379)
+ * Discrepancy in completion between prefixed and default namespaces. See[#311](https://github.com/angelozerr/lsp4xml/issues/311)
+ * When DTD is not found, XML is not checked for well-formedness. See[#167](https://github.com/redhat-developer/vscode-xml/issues/167)
+ * Formatter inserts spaces in empty lines. See[#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * vscode not reloading and revalidating XML/XSD files modified outside itself. See[#131](https://github.com/redhat-developer/vscode-xml/issues/131)
+
+### Performances
+
+ * Improve XML Scanner performance. See[#444](https://github.com/angelozerr/lsp4xml/issues/444)
+ * Use CompletableFuture to load DOMDocument. See[#439](https://github.com/angelozerr/lsp4xml/issues/439)
+ * Feedback from memory. See[#438](https://github.com/angelozerr/lsp4xml/issues/438)
+ * Improve performance of TextDocument update (in async) with TreeLineTracker. See[#426](https://github.com/angelozerr/lsp4xml/issues/426)
```

Comment:
I would suggest "Improved `TextDocument` update (in async) performance with `TreeLineTracker`."